data/strings/types_strings.txt

```diff
@@ -8462,7 +8300,7 @@
     zh-Hant = 麵食
 
   [type.cuisine.oriental]
-    en = Far Eastern
+    en = Oriental
```

Comment:
In the US, the normal term that encompasses many countries is "Asian", except that Indian restaurants are not called Asian -- which I never realized was odd until just now.   The real question is what the OSM codepoint means.  If it more or less encompasses Chinese, Japanese, Vietnamese, Thai, then "Asian" is good.

Reply:
cuisine-asian is also a thing in OM/OSM, maybe they should be merged in another PR?

Reply:
East Asian seems good 👍